features/gammatone.py

```diff
@@ -29,27 +33,53 @@ def GammatoneJob(crp, gt_options=None, **kwargs):
 
 
 def gammatone_flow(
-    minfreq=100,
-    maxfreq=7500,
-    channels=68,
+    minfreq: int = 100,
+    maxfreq: int = 7500,
+    channels: int = 68,
     warp_freqbreak=None,
```

Comment:
```suggestion
    warp_freqbreak: Optional[int] = None,
```
This should be of same type as minfreq/maxfreq